hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/FSConfigToCSConfigConverter.java

```diff
@@ -412,6 +414,14 @@ private void emitDefaultMaxAMShare() {
           queueMaxAMShareDefault);
     }
   }
+
+  private void emitDefaultUserLimitFactor() {
+    capacitySchedulerConfig.setFloat(
+            CapacitySchedulerConfiguration.
+                    PREFIX + "root.default." + USER_LIMIT_FACTOR,
```

Comment:
There might be other queues besides the default one which also need to have the default user-limit-factor. Can you please address that?